README.md

```diff
@@ -128,6 +140,11 @@ async def aresponses(loop):
 
 ### Updating package on pypi
   - `make deploy`
+  
+### if not using makefile
```

Comment:
are you running on windows?

Reply:
no~using mac :)  I use `virtualenvwrapper` so just simply ignore it at the 1st step

tried it just now, seems awesome, let me remove the extra lines

Reply:
is `make` not installed in on mac by default? Once you activate your environment I would think that `make test` and `make lint` still worked?